### DISCUSSION TITLE
How do I remove the "On Default" text in the directory module?

### DISCUSSION THREAD

**Discussion by sgiddings-kunzleigh** [starship/starship#4418] [Q&A] [answered]
I have been looking all over for this and have not found an answer. I dislike the "On Default" text displayed in the prompt when my shell opens to my home directory. I have tried altering the `format` in the `directory` module but it persists. I have googled all over, checked GitHub issues, and scoured the Starship.rs website and cannot find an answer. I thought I found an answer in the bracketed preset TOML file but it has not helped. 

Here is my current code for the directory module:

```
[directory]
truncate_to_repo = true
truncation_length =  3
truncation_symbol = ".../"
format = '\[[$path]($style)[$read_only]($read_only_style)\] '
home_symbol = "~"
use_os_path_sep = true
```

And here is how it renders in my terminal:

![image](https://user-images.githubusercontent.com/72622131/193079411-f132e692-6967-41ed-9cdf-04e14aaf9961.png)

I feel like on an older computer I was able to remove this wording and only display the current working directory, but I cannot find how to do that anymore.

**Comment by chipbuster** [comment, marked as answer]
That doesn't look like it's the directory module. The blue tilde should be the directory module.

What does `starship explain` show you?

**Comment by sgiddings-kunzleigh** [comment]
Wow....

Thank you. I did not know that `starship explain` was there. It is NOT the directory module, somehow Terraform was in my prompt and that was it.

Thank you for helping me with that. I am grateful to have that command in my toolbox now.
